### PR TITLE
fix(tile): slotted content renders within the border

### PR DESCRIPTION
**Pull request by eriklharper**
**Related Issue:** #9358

## Summary

This PR fixes an issue where slotted image content would overflow beyond the boundary of a `calcite-tile`.  Slotted content is now set to take the full width of the container, regardless of its size on its own.  This will keep visual overflow from happening, and allow developers to control the sizing of tiles with simple CSS selectors targeting the `calcite-tile` element.

### CODE REVIEW

**Review by geospatialem — COMMENTED** [review]
Awesome! 🎉 

Looks **_fantastic_** with the both the [giraffe](https://codepen.io/mac_and_cheese/pen/gOvBqxy)🦒 and [bear](https://codepen.io/geospatialem/pen/GRaJdwE)🐻 samples.

image

image

**Review by driskull — APPROVED** [review]
👍

File: packages/calcite-components/src/components/tile/tile.stories.ts [hunk 1]
```diff
@@ -348,23 +348,23 @@ export const allVariants = (): string => html`
...
...YfTM0u6ovamLRDpLC/C197nhKbMc+ZQVpoVO4DnexHUekg1MK9U6vtqOXxEHp+0Z9FFP/VbX+EfGByB+76GZuf6hOfEXRq4Pp2KHMlbKeu1XEVNRuW4WudN/LpLHA0CPDWAJ5cNan+nvB8afhRQo6i2o+ptLXK8nupr4kslIb/jqG/BRx9//oz5zb7l9JIum7NU2RTTYA28TtYgKRe4I3PLa8z+IwwpsU+IjbURa3oOXvv6S/wfaLS4QKBSFlCksXC9dRO58pKzTIBUOujpAP2bnc9QeE51Y1KnyZFSSbdfzl9leVa11ubIN783txA8usTCZI5e9QFFHpdt7WHTnvOXaDOCPqadgo8JtbfyHQRpA5eEGN7RsraaQUIBYAAbqP0/tO+W5mcTUCslj8QNwVUATJAamJ4/1ml7Koe8G1hpYHr1J/ISdEXVDu1rvTVRwRt9vtsOR9NvnMPisz0Lb7R5TW9t8YKilV4oCQdRsAbX/Keb1VPEn3M6xim+SDk6JdDFuG16t7/Py85rckptjB0C7Ow4DoB5kTL5FlT4ur3dPZRYvUI8KJ/UnkOfzM9VwODShTWlTFlX5sebMeZMv6bR9R3Lt/TO1erWNbY9/wCDsNh0pItOmoVVFgo5fued46OMJtKNKkYTlbtjYQjrRkRsLR1oQsdDYR1oWgFMbCOtC0YhsdCFoqJJhCLaFohphC0WERO0NhHERpjSIN0NjoQtCg3BCFotoUFs8/wWAKb1CE2+2SG9lAvNHl2EoujO7Ahb2FgGcgX8Nzf8okJ42UmeziuCOM8KONCgKthpIBJHMEnfrwtL3E5VSxdPXTIUniV3GviQw8vKEIvIS4REweEwmFb6wmo63uQvgv0seJH9ZW57j2rvq3CjgpPAenC8ISSEiGhtbrL7J8TWYhEayfaBsQq33tfnx+cIQ8g+xMzrM9A0KN2BAO1lG1yBMbXpHVf9YQkhLsdsJhbnc8OMTM89FJe7omzbi4O/Lbb0hCOPL5B9jlhH72n3jfavf/dzIlBSympXr9xTF992N9KJ95ughCWsEVKaTK2aTjFtej03KMsTCUhSpjzZz8VR+bN+3IbSZCE9JGKjHg8s5OTcmEIQjEEIQgARYQgNMcqx5SEJyfcsR7HMrGlYsJKJzlFBaFoQkyDQWiQhBEQiwhAEELQhESYmmLaEIwaQWiWhCMif/9k="
```
[the remark this answers is not in the document]
Comment (jcfranco) [reply packages/calcite-components/src/components/tile/tile.stories.ts:351]: For consistency, my vote is on using `placeholderImage`.

**Comment by jcfranco** [comment]
Can you update the PR title to use the imperative, present tense per the [conventions](https://github.com/ionic-team/stencil/blob/main/CONTRIBUTING.md?rgh-link-date=2024-04-03T00%3A01%3A43Z#subject) we link to in our contributing doc?

File: packages/calcite-components/src/components/tile/tile.stories.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (eriklharper) [reply packages/calcite-components/src/components/tile/tile.stories.ts:351]: That looks way better, thanks for the tip!  I swear I learn about some new tool like this every PR :)

**Pull request merged**